src/app/app.component.ts

```diff
@@ -1,10 +1,37 @@
 import { Component } from '@angular/core';
+import { Subscription } from "rxjs";
+
+import { ArticlesService } from "./services/articles.service";
+import { Article } from "./models/article.model";
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'kristina-project';
+  constructor(private articlesService: ArticlesService) {}
+
+  articles: Article[] = [];
+  sub = new Subscription();
+
+  ngOnInit(): void {
+    this.getArticles();
+  }
+
+  ngOnDestroy (): void {
+    this.sub.unsubscribe();
```

Comment:
it is a rare case, but just try to use "?" for sub, it is like a best practice:
this.sub?.unsubscribe();